android-sdk/src/main/java/org/sagebionetworks/bridge/android/BridgeConfig.java

```diff
@@ -98,6 +98,11 @@ public int getAppVersion() {
     }
 
     @NonNull
+    public String getAppVersionName() {
+        return applicationContext.getResources().getString(R.string.osb_app_version_name);
```

Comment:
Good point. For Bridge, version name needs to be in a specific format, but I don't see a problem with documenting that and using the standard versionCode and versionName